hoomd/hpmc/integrate.py

```diff
@@ -133,16 +134,23 @@ def __init__(self, seed, d, a, translation_move_probability, nselect):
         typeparam_fugacity = TypeParameter('depletant_fugacity',
                                            type_kind='particle_types',
                                            param_dict=TypeParameterDict(
-                                               0., len_keys=1))
+                                               0., len_keys=2)
+                                           )
+
+        typeparam_ntrial = TypeParameter('depletant_ntrial',
```

Comment:
Add documentation for `depletant_ntrial` to the class docstring.

Reply:
done